src/Sarif.FunctionalTests/SarifValidatorTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Validation;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public class SarifValidatorTests
+    {
+        public const string TestDirectory = "SarifConverterTestData";
+        public const string JsonSchemaFile = "Sarif.schema.json";
+
+        private readonly string _jsonSchemaFilePath;
+
+        public SarifValidatorTests()
+        {
+            _jsonSchemaFilePath = Path.Combine(Environment.CurrentDirectory, JsonSchemaFile);
+        }
+
+        [Theory(Skip = "https://github.com/Microsoft/sarif-sdk/issues/108")]
+        [MemberData(nameof(TestCases))]
+        public void Sample_tool_output_files_conform_to_SARIF_schema(string inputFile)
+        {
+            IEnumerable<JsonError> errors = Validator.ValidateFile(inputFile, JsonSchemaFile);
```

Comment:
The guts of it: we use the `Validator` class, now refactored into the `Sarif.Validators` project, to validate the test files.